foundry.toml

```diff
@@ -2,6 +2,8 @@
 src = "src"
 out = "out"
 libs = ["lib"]
+build_info = true
+extra_output = ["storageLayout"]
```

Comment:
@gpylypchuk Since we don't use https://github.com/OpenZeppelin/openzeppelin-foundry-upgrades anymore do we need this settings (`build_info`, `extra_output`)?